### DISCUSSION TITLE
FLAC 1.4.1 just came out

### DISCUSSION THREAD

**Discussion by Thaldan** [enzo1982/freac#376] [Audio Format Discussion]
Hi,

A new FLAC version just came out two days ago, have you any planes to add it?

Cordially

**Comment by enzo1982** [comment]
fre:ac's FLAC will be updated to version 1.4.1 soon. There are no changes relevant to fre:ac in that release, though, and no changes to the codec itself.

**Reply by enzo1982** [reply]
No relevant changes compared to FLAC 1.4.0 which already is available in fre:ac continuous builds that is, of course.

**Reply by juanpc2018** [reply]
> fre:ac's FLAC will be updated to version 1.4.1 soon. There are no changes relevant to fre:ac in that release, though, and no changes to the codec itself.

FLAC latest version is 1.4.2

Freac Method is Absurd / Insane / Beyond Crazy...

FFE method is better:
https://flacfrontend.sourceforge.net/
simply download latest:
https://xiph.org/flac/download.html
https://ftp.osuosl.org/pub/xiph/releases/flac/
Copy & Paste to folder, done.
works in Wine 5.9 PlayOnMac 4.3.3 _ 32-bit

but lacks the multi-core parallel processing of XLD method: "1-core per file"
https://tmkk.undo.jp/xld/index_e.html
that method is amazing, but does Not sound as good as FFE 1.4.x
 
1.4.2 sound better, and is faster than 1.3.3

havent done a bit-by-bit comparison,
CRC MD5 SHA check sum,
maybe it sounds better because its more optimized,
similar to mp4 h264 video that has [X]web optimized in handbrake.fr
allows to play back faster the same file, plays/decode better.

OSX 
$ Brew install flac 
works in HighSierra.

FLACCL is nice for large files.

VLC OSX Method is time consuming but sounds nice, RT playback does Not sound as good as Decoding to .wav
with bitrate of 100000

OSX does Not have a file to put inside the .app
/Applications/freac.app/Contents/Resources/codecs/cmdline/flac.sh

bash shell script is Not detected.

but brew creates a file here:
/usr/local/Cellar/flac/1.4.2/bin/flac

creating a folder & copy does Not work, Not detected.
.../codecs/cmdline/flac1.4.2/flac
.../codecs/cmdline/flac1.4.2/metaflac

**Reply by enzo1982** [reply]
FLAC 1.4.2 has been available in fre:ac continuous builds for one month already: https://github.com/enzo1982/freac/releases/continuous

> 1.4.2 sound better, and is faster than 1.3.3

FLAC is a lossless format. It must always sound the same regardless of the version. If it does not, something is broken on your end (hardware issue, flaky audio gear, ...).
